frontend/src/app/components/FeaturedExperiment/index.js

```diff
@@ -5,7 +5,6 @@ import React from "react";
 import { Link } from "react-router-dom";
 
 import { experimentL10nId } from "../../lib/utils";
-import ExperimentTourDialog from "../ExperimentTourDialog";
```

Comment:
I moved the tour up to the homepage. It was more convenient to handle both the email dialog and tour up there.